docs/api/v1/vm.md

```diff
@@ -0,0 +1,57 @@
+## /vm
+
+```
+vm = {
+  id: int,
+  cores: int,
+  ram: Bytes (int),
+  customer_id: int,
+  cputime_limit: int,
+  uuid: UUID (String),
+  state: {
+    id: StateID (int),
+    name: String,
+    description: String,
+  },
+  node_method: {
+    id: int,
+    virt_method: {
+      id: int,
+      name: string,
```

Comment:
`vm.node_method.virt_method.name` should not be listed here since `virt_method` will get a dedicated endpoint and then documentation of this value would be duplicated. `.id` should remain because it is referenced in the text in the notes section.